activities/membership-growth/membership-proposal-template.md

```diff
@@ -0,0 +1,116 @@
+* community for your codebases - understanding how to collaborate better and more openly with other organizations
+* product packaging of your codebases - understanding how to better market and promote your codebases
+* support of your codebases - understanding how to improve your collaboration with the market
+* [assessment of your codebases](https://about.publiccode.net/activities/codebase-stewardship/criteria-for-codebase-stewardship.html) for codebase stewardship
+
+As an association, our resources are allocated across members. Our support for each member is subject to availability.
+
+## Proposed activities
+
+[public organization] and the Foundation for Public Code will work together to identify appropriate codebases for codebase auditing, certification and stewardship activities.
+
+The Foundation for Public Code’s activities will include:
+
+* audits to help comply with the Standard for Public Code on every pull request
+* training and workshops to develop better reusable codebases as well as working with the community
+* events and regular calls to integrate a community
+* connecting relevant members of this or other communities in order to increase uptake and contributions
+* storytelling and marketing of the project based on best practices and research to support reuse
+* provide prospective reusers and contributors with a helpdesk to answer their questions on issues ranging from installations to procurement
+* help create budgets and frame costs for [public organization]’s activities outlined below
+
+[public organization] must commit time and budget to:
+
+* provide in-kind contributions of staff time to help us develop the Foundation for Public Code’s governance model, membership and financial structure
+* make all code and policy contributions compliant with the Standard
+* make selected code and policy contributions, as well as the entire projects, reusable by others
+* partake in the community in order to further development and help others reuse the codebase
+* support in product marketing and access to other organizations in order to enable reuse
+* organizational knowledge on procurement and technology to help others reuse the codebase
+* provide structural feedback on how to improve codebase stewardship activities as well as the Standard for Public Code
+
+## Next steps
+
+1. Agree membership dues.
+2. Review the [membership contract](https://about.publiccode.net/activities/membership-growth/membership-contract.html) and the [articles of association](https://about.publiccode.net/organization/articles-of-association.html), and let us know whether your organization is able to agree to these.
+3. Organize follow up meeting with [name] to finalize this proposal and clarify any questions about the contract.
+4. Sign contract and send back to [name].
+5. [bespoke]
+
+## Contact
+
+Laura Scheske, membership developer
+laura@publiccode.net
++31 202 444 561
```

Comment:
Do you have a VOIP number yet? Would you like one?
[Please add your number](https://about.publiccode.net/activities/communication/phone-numbers.html) here!

Reply:
I do not ! I would like one!

Reply:
Changed it to the general number for now, while mine is being set up.